staging/src/k8s.io/api/admissionregistration/v1alpha1/types.go

```diff
@@ -381,28 +340,20 @@ type MatchResources struct {
 }
 
 // NamedRuleWithOperations is a tuple of Operations and Resources with ResourceNames.
+// +structType=atomic
 type NamedRuleWithOperations struct {
 	// ResourceNames is an optional white list of names that the rule applies to.  An empty set means that everything is allowed.
 	// +listType=atomic
 	// +optional
 	ResourceNames []string `json:"resourceNames,omitempty" protobuf:"bytes,1,rep,name=resourceNames"`
 	// RuleWithOperations is a tuple of Operations and Resources.
+	// +listType=atomic
```

Comment:
since this isn't on the list itself, I don't think this does anything... probably best to drop it to avoid being misleading